recipes/ovs_offload/1_virt_ovs_vxlan.py

```diff
@@ -64,10 +64,10 @@ def get_vxlan_dev(host):
     cmd = host.run("ls /sys/class/net")
     out = cmd.out().strip().split()
 
-    if vxlan_dev in out:
-        return vxlan_dev
-    elif vxlan_dummy in out:
+    if vxlan_dummy in out:
         return vxlan_dummy
+    elif vxlan_dev in out:
+        return vxlan_dev
     else:
```

Comment:
looks like a workaround because you have both real vxlan and dummy vxlan our custom ovs version did. that ovs was planned for centos7.2 only which there the kernel doesnt create vxlan interface so we did.